earthly/python/Earthfile

```diff
@@ -0,0 +1,34 @@
+# Common Python UDC's and Builders.
+VERSION 0.7
+FROM python:3.11-bullseye
+
+# Common Poetry setup
+# Parameters:
+#   opts      : Options passed to the `poetry install` command.
+#   extra_file: An extra file to copy into the poetry path.
+POETRY_SETUP:
+    COMMAND
+    ARG opts
+    ARG extra_file
```

Comment:
I think we can make this `extra_files` and have it be a space-separated list of filenames.

Reply:
I tried that and it didn't work at all.  The argument was taken as a single file named: "file1 file2 file3".
It is my preference that it work that way, so if you can suggest how to make it work, more than happy to add it.

Reply:
That's too bad, probably some shell escaping going on. We can leave it for now and come back to it later if it really is a problem.